packages/image_picker/example/lib/main.dart

```diff
@@ -33,52 +34,207 @@ class MyHomePage extends StatefulWidget {
+          FloatingActionButton(
+            onPressed: () {
+              isVideo = false;
+              _onImageButtonPressed(ImageSource.gallery);
+            },
+            heroTag: 'image0',
             tooltip: 'Pick Image from gallery',
             child: const Icon(Icons.photo_library),
           ),
-          new Padding(
+          Padding(
             padding: const EdgeInsets.only(top: 16.0),
-            child: new FloatingActionButton(
-              onPressed: () => _onImageButtonPressed(ImageSource.camera),
+            child: FloatingActionButton(
+              onPressed: () {
+                isVideo = false;
+                _onImageButtonPressed(ImageSource.camera);
+              },
+              heroTag: 'image1',
               tooltip: 'Take a Photo',
               child: const Icon(Icons.camera_alt),
             ),
           ),
+          Padding(
+            padding: const EdgeInsets.only(top: 16.0),
+            child: FloatingActionButton(
+              backgroundColor: Colors.red,
+              onPressed: () {
+                isVideo = true;
+                _onImageButtonPressed(ImageSource.gallery);
+              },
+              heroTag: 'video0',
+              tooltip: 'Pick Video from gallery',
+              child: const Icon(Icons.video_library),
+            ),
+          ),
+          Padding(
+            padding: const EdgeInsets.only(top: 16.0),
+            child: FloatingActionButton(
+              backgroundColor: Colors.red,
+              onPressed: () {
+                isVideo = true;
+                _onImageButtonPressed(ImageSource.camera);
+              },
+              heroTag: 'video1',
+              tooltip: 'Take a Video',
+              child: const Icon(Icons.videocam),
+            ),
+          ),
         ],
       ),
     );
   }
 }
+
+class AspectRatioVideo extends StatefulWidget {
+  final VideoPlayerController controller;
+
+  AspectRatioVideo(this.controller);
+
+  @override
+  AspectRatioVideoState createState() => new AspectRatioVideoState();
+}
+
+class AspectRatioVideoState extends State<AspectRatioVideo> {
+  VideoPlayerController get controller => widget.controller;
+  bool initialized = false;
+
+  VoidCallback listener;
+
+  @override
+  void initState() {
+    super.initState();
+    listener = () {
+      if (!mounted) {
+        return;
+      }
+      if (initialized != controller.value.initialized) {
+        initialized = controller.value.initialized;
+        setState(() {});
+      }
+    };
+    controller.addListener(listener);
+  }
+
+  @override
+  Widget build(BuildContext context) {
+    if (initialized) {
+      final Size size = controller.value.size;
+      return new Center(
+        child: new AspectRatio(
+          aspectRatio: size.width / size.height,
+          child: new VideoPlayer(controller),
+        ),
+      );
+    } else {
+      return new Container();
+    }
+  }
+}
```

Comment:
Please add empty line at the end of the file.

Reply:
When I add a empty line it failed the format test?